packages/headless/src/api/platform-client.ts

```diff
@@ -124,76 +124,48 @@ export class PlatformClient {
   }
 }
 
-interface URLOptions<E extends PlatformEnvironment> {
-  environment?: E;
-  region?: Extract<PlatformCombination, {env: E}>['region'];
-  multiRegionSubDomain?: string;
-}
-
-function coveoCloudURL<E extends PlatformEnvironment>(
-  subdomain: string,
-  options?: URLOptions<E>
+export function getOrganizationEndpoint(
```

Comment:
Are we missing jsdoc on these exported functions?

Reply:
Fixed